content/content.go

```diff
@@ -53,6 +65,24 @@ type (
 	GlobalRuleConfig = types.GlobalRuleConfig
 )
 
+var (
+	// MandatoryRuleWideContentFiles are mandatory files that MUST be on rule plugin or error key level
+	MandatoryRuleWideContentFiles = []string{GenericMarkdown, ReasonMarkdown}
+
+	// SharedContentFiles MAY be on either the plugin level or the error key level
+	SharedContentFiles = []string{GenericMarkdown, ReasonMarkdown, SummaryMarkdown, ResolutionMarkdown, MoreInfoMarkdown}
+
+	// RulePluginMandatoryContentFiles are mandatory on the plugin level
+	RulePluginMandatoryContentFiles = []string{PluginYAML}
+	// RulePluginContentFiles are all files to look for on rule plugin level
```

Comment:
nitpick -prob. empty line make it more readable

Reply:
done, thank you